notifications_utils/formatters.py

```diff
@@ -213,6 +220,10 @@ def strip_characters_inserted_to_force_newlines(value):
     )
 
 
+def tweak_dvla_list_markup(value):
```

Comment:
@kentsanggds Added a specific test in 9d81abc8e144a7fd7ce132674c190dda652a7070. Think it’s still worth having the one that’s more of an integration test because this is so closely related to what markup we generate for lists.